.typos.toml

```diff
@@ -5,3 +5,5 @@ inout = "inout"
 ocur = "ocur"
 bigsur = "Big Sur"
 Smoot = "Smoot"
+ba = "ba"
+Datas = "Datas"
```

Comment:
Nit: TBH, since it is so unlikely we'd use the identifiers again I'd probably just use different data in the test and renamed the top-level unit to e.g., `Messages` than add these here.

Reply:
Would also require changing input here.